docs/usage/web-component.md

```diff
@@ -0,0 +1,52 @@
+# Web Component
+
+If you are not using React you may want to use the `@asyncapi/web-component` component as a plain web component in any other web framework of your choice or as an element of a static HTML webpage. This is achieved by making use of [web-react-components](https://www.npmjs.com/package/web-react-components).
+
+## Overview
+
+Read the document to find out how to install dependencies and attach `web-component` in your project.
+
+## Installation
+
+Run this command to install the component in your project:
+
+```sh
+npm install --save @asyncapi/web-component
+```
+
+Check out this simple sandbox application that uses the Web Component in Angular project:
+
+[![Edit asyncapi-web-component-in-action](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/asyncapi-web-component-in-action-l652x)
+
+## Usage
```

Comment:
```suggestion
## Examples
```

sometimes you write examples sometimes usage, stick to one

Reply:
I prefer `Usage` :)